.travis.yml

```diff
@@ -3,5 +3,7 @@ language: python
 python:
 - "2.7"
 - "3.6"
+
```

Comment:
This BOTO_CONFIG thing is new, some kind of strange BOTO issue on travis as it uses google compute engine and the GCE nodes have some kind of boto config on them.  The GCE boto config breaks our build.

This little nugget fixes travis.